[2025-04-04T19:17:19.215+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-04-04T19:17:19.244+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: e_commerce_dw_etl.transform.transform_dim_drivers manual__2025-04-04T19:17:15.548172+00:00 [queued]>
[2025-04-04T19:17:19.251+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: e_commerce_dw_etl.transform.transform_dim_drivers manual__2025-04-04T19:17:15.548172+00:00 [queued]>
[2025-04-04T19:17:19.252+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-04-04T19:17:19.262+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): transform.transform_dim_drivers> on 2025-04-04 19:17:15.548172+00:00
[2025-04-04T19:17:19.271+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'e_commerce_dw_etl', 'transform.transform_dim_drivers', 'manual__2025-04-04T19:17:15.548172+00:00', '--job-id', '42', '--raw', '--subdir', 'DAGS_FOLDER/ETL_dag.py', '--cfg-path', '/tmp/tmpmdipx7tx']
[2025-04-04T19:17:19.274+0000] {standard_task_runner.py:91} INFO - Job 42: Subtask transform.transform_dim_drivers
[2025-04-04T19:17:19.274+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=315) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-04T19:17:19.275+0000] {standard_task_runner.py:63} INFO - Started process 319 to run task
[2025-04-04T19:17:19.317+0000] {task_command.py:426} INFO - Running <TaskInstance: e_commerce_dw_etl.transform.transform_dim_drivers manual__2025-04-04T19:17:15.548172+00:00 [running]> on host 3b69888ed614
[2025-04-04T19:17:19.397+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='e_commerce_dw_etl' AIRFLOW_CTX_TASK_ID='transform.transform_dim_drivers' AIRFLOW_CTX_EXECUTION_DATE='2025-04-04T19:17:15.548172+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-04T19:17:15.548172+00:00'
[2025-04-04T19:17:19.398+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-04-04T19:17:19.453+0000] {base.py:84} INFO - Using connection ID 'postgres_default' for task execution.
[2025-04-04T19:17:19.466+0000] {base.py:84} INFO - Using connection ID 'postgres_default' for task execution.
[2025-04-04T19:17:19.524+0000] {logging_mixin.py:188} INFO - Lỗi khi insert dữ liệu: duplicate key value violates unique constraint "dim_drivers_pkey"
DETAIL:  Key (driverid)=(1) already exists.
[2025-04-04T19:17:19.526+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-04-04T19:17:19.527+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-04-04T19:17:19.536+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=e_commerce_dw_etl, task_id=transform.transform_dim_drivers, run_id=manual__2025-04-04T19:17:15.548172+00:00, execution_date=20250404T191715, start_date=20250404T191719, end_date=20250404T191719
[2025-04-04T19:17:19.569+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-04-04T19:17:19.585+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-04T19:17:19.587+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
